### PR TITLE
update notes for dmctl

### PR DESCRIPTION
**Pull request by lance6716**
### What is changed, added or deleted? (Required)

 add notes to clarify https://github.com/pingcap/dm/issues/885

### Which DM version(s) do your changes apply to? (Required)

- [x] master (the latest development version, including v2.0 changes for now)
- [x] v1.0 (TiDB DM 1.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s): https://github.com/pingcap/dm/issues/885

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Have version specific changes 
- [ ] Might cause conflicts

### CODE REVIEW

File: zh/dmctl-introduction.md [hunk 1]
```diff
@@ -14,6 +14,10 @@ dmctl 同时支持交互模式和命令模式。
 
 进入交互模式，与 DM-master 进行交互：
 
+> **注意：**
+>
+> 交互模式下不具有 bash 的特性，比如不需要通过引号传递字符串参数而应当直接传递
```
Comment (ireneontheway) [inline zh/dmctl-introduction.md:19]:
```suggestion
> 交互模式下不具有 bash 的特性，比如不需要通过引号传递字符串参数而应当直接传递。
```

**Comment by csuzhangxc** [comment]
@TomShawn two LGTM now, PTAL

**Pull request merged**